### PR TITLE
Added pytypes

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

File: recipes/pytypes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{% set sha256 = "20ae0aa741c63126eb697c41ad576239e397436d9fba93934622d0fd7b20dfd6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # url: https://github.com/simplejson/{{ name }}/archive/{{ version }}.tar.gz
```
Comment (CJ-Wright) [inline recipes/pytypes/meta.yaml]: please remove this

File: recipes/pytypes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - python
+    - typing
+
+test:
+  imports:
+    - pytypes
```
Comment (CJ-Wright) [inline recipes/pytypes/meta.yaml]: Would it be possible to put in some more tests? (More import tests are ok)

Comment (mariusvniekerk) [reply recipes/pytypes/meta.yaml]: That import basically imports everything in the module, not a very large one

**Comment by mariusvniekerk** [comment]
ping @CJ-Wright

**Pull request merged**